source/language/classical.rst

```diff
@@ -427,6 +427,190 @@ Terminating the program early
 The statement ``end;`` immediately terminates the program, no matter what scope
 it is called from.
 
+
+The Switch statement
+--------------------
```

Comment:
Nitpick: If looks like the heading level for the `switch` statement is incorrect. I assume the intention is to have it within the `Looping and branching` section, but instead it is a sibling.

On the other hand, the heading level for `Looping and branching` also seems odd -- it's currently inside `Low-level classical instructions` which makes it not findable in the table of contents on the left.

Perhaps `Looping and branching` should be moved up a level? I'm happy to make a separate PR for that if people like.